### ISSUE TITLE
'flutter packages get' failing after adding flutter_launcher_icons dependecy

### ISSUE THREAD

**Issue by kkaun** [fluttercommunity/flutter_launcher_icons#85]
Recently I caught this when trying to install flutter_launcher_icons: ^0.7.0:
```
[dart_space] flutter packages get
Running "flutter packages get" in dart_space...                 
Git error. Command: git clone C:\Users\Кира\AppData\Roaming\Pub\Cache\git\cache\dart_config-d2db83fa92bb229094789c5489c5e2d3cc175814 C:\Users\Кира\AppData\Roaming\Pub\Cache\git\dart_config-a7ed88a4793e094a4d5d5c2d88a89e55510accde

Cloning into 'C:\Users\РљРёСЂР°\AppData\Roaming\Pub\Cache\git\dart_config-a7ed88a4793e094a4d5d5c2d88a89e55510accde'...

"git-upload-pack 'C:\Users\РљРёСЂР°\AppData\Roaming\Pub\Cache\git\cache\dart_config-d2db83fa92bb229094789c5489c5e2d3cc175814'": git-upload-pack 'C:\Users\РљРёСЂР°\AppData\Roaming\Pub\Cache\git\cache\dart_config-d2db83fa92bb229094789c5489c5e2d3cc175814': No such file or directory

fatal: Could not read from remote repository. 
Please make sure you have the correct access rights and the repository exists.

pub get failed (69) -- attempting retry 1 in 1 second...
```
And so on.

I tried a lot of different libs and saw nothing similar to this. Moreover, when I remove flutter_launcher_icons from dependecies `packages get` runs succesfully again.

BTW I don't use proxy or anything like this.

I doubt that this issue is quite similar to this: https://github.com/fluttercommunity/flutter_launcher_icons/issues/68 , but who knows. Maybe someone had packaging issue exactly like this one before? Thanks in advance.

**Comment by MarkOSullivan94** [comment]
Hey @kkaun thanks for reporting this, it's the first time I've been made aware of this issue. 

I did a quick search, dart_config is a dependency which is used in this project, I used to use a hosted version of it on pub before having to switch to my own forked version of dart_config whenever the one on pub was no longer working for us (when we switched to Dart 2).

https://github.com/MarkOSullivan94/dart_config/commit/a7ed88a4793e094a4d5d5c2d88a89e55510accde

That's the commit there and it's a public repo so I don't know why there's an issue with not having access like it states in the error message.

Might be best to report it to pub GitHub page to see if they can help: https://github.com/dart-lang/pub/issues
